Formula/x/xk6.rb

```diff
@@ -23,11 +23,10 @@ class Xk6 < Formula
   depends_on "go"
 
   def install
-    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/xk6"
+    system "go", "build", *std_go_args(ldflags: "-s -w -X go.k6.io/xk6/internal/cmd.version=#{version}"), "."
   end
 
   test do
-    str_build = shell_output("#{bin}/xk6 build")
-    assert_match "xk6 has now produced a new k6 binary", str_build
+    assert_match "xk6 version #{version}", shell_output("#{bin}/xk6 version")
```

Comment:
any chance to test other commands other than `version`?

Reply:
> This reduces the robustness of the test due to network uncertainties.

This has never been an issue across any of the Homebrew test blocks.

Please do not remove the build test. In my experience as a homebrew maintainer, testing something that people will likely use is the only way to catch real world breakage.

Reply:
Done, I put the build test back.